### ISSUE TITLE
TFlite runtime wheel not supported

### ISSUE THREAD

**Issue by albydnc** [tensorflow/tensorflow#41098]
**System information**
- Ubuntu 20.04 with linux kernel 5.7.1
- Bare Python 3.8.2 and Conda Env with Python 3.7.6

**The problem**
 
When trying to install *tflite_runtime* I get the following error:
```
$ pip3 install https://dl.google.com/coral/python/tflite_runtime-2.1.0.post1-cp37-cp37m-linux_x86_64.whl
ERROR: tflite_runtime-2.1.0.post1-cp37-cp37m-linux_x86_64.whl is not a supported wheel on this platform.
```
I've tried a bare installation in Python 3.8.2 and in a conda environment with python 3.7.6. Same error for each try.

**Comment by Saduf2019** [comment]
@albydnc 
Could you please check if you are using 64-bit version of Python?
TensorFlow is tested and supported on 64-bit systems, for more information please check the [system requirements here](https://www.tensorflow.org/install/pip#system-requirements). Thanks!


Tf  2.1 does not support Python 3.8 please refer to [this issue](https://github.com/tensorflow/tensorflow/issues/39768#issuecomment-632799481).
Please refer to these links:
[link](https://github.com/tensorflow/tensorflow/issues/40890#issuecomment-651037848) [link](https://stackoverflow.com/questions/33622613/tensorflow-installation-error-not-a-supported-wheel-on-this-platform) [link1](https://github.com/Qengineering/TensorFlow-Raspberry-Pi/issues/1) [link2](https://github.com/tensorflow/tensorflow/issues/9722) [link3](https://www.tensorflow.org/lite/guide/python)

**Comment by albydnc** [comment]
@Saduf2019 
Thanks for your answer. Both python are 64-bit. I've assumed TF 2.1 wasn't available for 3.8 as you reported.
In fact, I was mainly trying to install it on a conda env with Python 3.7.6 64-bit. From conda navigator I've installed and used TF without problems.
The issue comes with a new env where I just wanted the TFlite_runtime package and I get the error reported above.

**Comment by ymodak** [comment]
Can you try upgrading your pip to latest version and installing again?
